Bottom: eda691a4d1cbd65c62247ffa73dc3c217da09817
Top:    1a1af736fb7a8416fa8949f0a724f57ebe06768e
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2015-01-10 16:15:33 +0000

w32: Update gd recipie to new version and download location for libgd


---

diff --git a/w32/recipes/gd.recipe b/w32/recipes/gd.recipe
index c9588a3..a67bc1b 100644
--- a/w32/recipes/gd.recipe
+++ b/w32/recipes/gd.recipe
@@ -1,7 +1,7 @@
-name=gd
-version=2.0.35
+name=libgd
+version=2.1.0
 release=1
-download=http://www.libgd.org/releases
+download=https://bitbucket.org/libgd/gd-libgd/downloads
 
 configure_opt="
   --disable-static
